src/fileformats/ocd_file_format.cpp

```diff
@@ -45,6 +72,16 @@ OcdFileFormat::OcdFileFormat()
 	// Nothing
 }
 
+OcdFileFormat::OcdFileFormat(quint16 version)
+: FileFormat { MapFile, idForVersion(version),
```

Comment:
Both these entries and "OCAD" from default constructor (line 69) are presented in the Save as... dialog as possible file types. Only code inspection reveals that "OCAD" means "save with v9 or the version from which the map was loaded". A more descriptive text is needed.

Reply:
Proposal: "OCAD (auto-selected version)"

Details might go to a manual page on file formats (or OCD format support).